buildkite/run_check_deploy_docker_build.sh

```diff
@@ -4,20 +4,10 @@
 # built successfully. This check is done in the buildkite-assets repo to prevent
 # an unbuildable Dockerfile from being deployed to buildkite instances.
 
-DOCKER_IMAGE_NAME=check-heroku-deploy-image
+echo "--- Building and testing with panorama_command.sh"
+HEROKU_DEPLOY_PATH=deploy/heroku-deploy \
+  HEROKU_DEPLOY_IMAGE_NAME=check-heroku-deploy-image \
```

Comment:
This lets us build the local, updated Docker container in `panorama_command.sh` with a different tag name than the one we use for real Buildkite builds (`heroku-deploy-img`) so the one we build for testing in this repo doesn't stomp on the one the agents use for real in other repos. It probably wouldn't be an issue, because if the two `Dockerfile`s are different, `panorama_command.sh` would re-build the bootstrapped version, but there could be a race condition there (e.g. if the tag gets replaced after `panorama_command.sh` has built but before it runs its command in the container) and it seems better safe than sorry.